src/main/java/com/anqit/spanqit/core/Prefix.java

```diff
@@ -33,7 +33,7 @@ public Iri iri(String localName) {
 
 	@Override
 	public String getQueryString() {
-		return PREFIX + " " + label + ": " + iri.getQueryString();
+		return PREFIX + " " + iri.getQueryString();
 	}
```

Comment:
Hi @vladimir-chudik, thanks for the contrubtion, though I'm not sure I agree with this change. This class is supposed to represent a prefix declaration as described here by https://www.w3.org/TR/2013/REC-sparql11-query-20130321/#prefNames, or more formally in the grammar: https://www.w3.org/TR/2013/REC-sparql11-query-20130321/#rPrefixDecl

Is there a use case you have for printing just the iri without the label?

Reply:
Hi @anqit , yes, I have the use:

Let's say I have Sparql query:
PREFIX owl: <http://www.w3.org/2002/07/owl#>
PREFIX myNamespace: <...>
INSERT DATA
{
  myNamespace:Johny a myNamespace:Person
}

Then If I want to produce Prefix part, prefix.getQueryString() returns:
"PREFIX owl: owl: <http://www.w3.org/2002/07/owl#>" because of the second print of the "label" in iri.getQueryString. 
I am using iri.getQueryString to generate "myNamespace:Johny" which I then use in TriplePattern query, so I wouldn't delete the label from there.

I might be using it wrong, thanks for any recommendation.

Reply:
Do you mind sharing the code that is producing "PREFIX owl: owl: http://www.w3.org/2002/07/owl#"? 
Also, have you checked out the spanqit-examples project by any chance? https://github.com/anqit/spanqit-examples/blob/master/src/main/java/com/anqit/spanqit/examples/Section2.java in particular might have some example usages that might help.

Reply:
Do you mind sharing the code that is producing "PREFIX owl: owl: http://www.w3.org/2002/07/owl#"?
Also, have you checked out the spanqit-examples project by any chance? https://github.com/anqit/spanqit-examples/blob/master/src/main/java/com/anqit/spanqit/examples/Section2.java in particular might have some example usages that might help.

Reply:
Hi, this is the part in my code:
        Prefix owlPrefix = Spanqit.prefix("owl", null);
        owlPrefix = Spanqit.prefix("owl", owlPrefix.iri("<http://www.w3.org/2002/07/owl#>"));
        logger.info(owlPrefix.getQueryString());

Example says:
	Prefix foaf = Spanqit.prefix("foaf", iri(FOAF_NS));

However, calling just
        Prefix owlPrefix = Spanqit.prefix("owl", iri("<http://www.w3.org/2002/07/owl#>"));
is not possible, because iri statement itself is not recognized without initialization. 
I am using Spring Boot in IntelliJ, Java 8.

Reply:
@vladimir-chudik, if you update this branch now, there are a couple of `Rdf.iri()` static methods you can use to create Iri's now. I think this will fix your problem! Let me know if you're still having issues.